src/components/Modal/AddName/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import {
+	ButtonsWrapper,
+	CancelButton,
+	CloseIconDiv,
+	Input,
+	InputWrap,
+	Label,
+	ModalTitle,
+	Wrapper,
+} from "./styles";
+import { ErrorMessage, StartButton } from "../../CategoryCard/styles";
+import { useQuizData } from "../../../context/quizContext";
+import { CLOSE__MODAL, SET__PLAYER__NAME } from "../../../constants";
+import { ReactComponent as CloseIcon } from "../../../icons/close-icon.svg";
+import { useHistory } from "react-router";
+
+function AddName() {
+	const { dispatch } = useQuizData();
+	const [name, setName] = useState("");
+	const [isNameActive, setIsNameActive] = useState(false);
+	const [message, setMessage] = useState("");
+	const history = useHistory();
+	const onChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+		setName(e.target.value);
+		if (e.target.value.length > 0) {
```

Comment:
store e.target.value in a variable, don't repeat the same again

Reply:
done